index.html

```diff
@@ -521,7 +521,7 @@ <h3>
             #communi
           </tt>
           on
-          <a href="https://www.freenode.net">Freenode</a>.
+          <a href="https://libera.chat/">libera.chat</a>.
```

Comment:
You spelled Libera.chat wrong, names are capitalized.

Reply:
fixed. https://libera.chat/about/  says it is "Libera Chat" so I went with this.